controllers/dataprotection/utils.go

```
@@ -225,6 +225,31 @@ func getCluster(ctx context.Context,
 	return cluster
 }
 
+// listObjectsOfClusterWithErrorIgnored list the objects and will ignore the error.
+func listObjectsOfClusterWithErrorIgnored(ctx context.Context,
+	cli client.Client,
+	cluster *appsv1alpha1.Cluster,
+	object client.ObjectList) client.ObjectList {
+	labels := constant.GetClusterWellKnownLabels(cluster.Name)
+	if err := cli.List(ctx, object, client.InNamespace(cluster.Namespace), client.MatchingLabels(labels)); err != nil {
+		return nil
```

don't ignore the error